ui-tests/cypress/support/pageObjects/content/assets/AssetsPage.js

```diff
@@ -49,114 +85,87 @@ export default class AssetsPage extends AdminContent {
   }
 
   getFilterResultInfo() {
-    return this.getAssetsBody()
+    return this.get()
                .find(this.resultInfo);
   }
 
   getFilterResultItemCount() {
     return this.getFilterResultInfo()
+               .find(`#search > .pager`)
                .find(this.resultInfoItemCount);
   }
 
-  getTable() {
-    return this.getAssetsBody()
-               .find(htmlElements.table);
-  }
-
   getTableRows() {
-    return this.getTable()
-               .children(htmlElements.tbody)
-               .children(htmlElements.tr);
+    return this.getAssetsBody();
```

Comment:
Where is this method declared? Why do you need a method which just return another method call?